Datasnap/Classes/Event Properties/Identifier.m

```diff
@@ -46,8 +47,25 @@ - (NSDictionary*)convertToDictionary
         @"mobile_device_google_advertising_id" : self.mobileDeviceGoogleAdvertisingId ? self.mobileDeviceGoogleAdvertisingId : [NSNull null],
         @"mobile_device_google_advertising_id_opt_in" : self.mobileDeviceGoogleAdvertisingIdOptIn ? self.mobileDeviceGoogleAdvertisingIdOptIn : [NSNull null],
         @"unknown" : self.unknown ? self.unknown : [NSNull null],
-        @"facebook_uuid" : self.facebookUuid ? self.facebookUuid : [NSNull null]
+        @"facebook_uuid" : self.facebookUuid ? self.facebookUuid : [NSNull null],
+        @"hashed_email" : [self sha1:self.hashedEmail]
```

Comment:
I'm still not sold on the convertToDictionary methods throughout.  However, since they are in place, for readability's sake it would be nice of dict keys were set in alphanumeric order.

Also, it appears that for all other keys there is a check to see if a property exists, why not for self.hashedEmail?